SwiftSyntaxCLI/Sources/swift-syntax-cli/common/Paths.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  File.swift
+//
+//
+//  Created by Kim de Vos on 26/04/2023.
+//
+
+import Foundation
+
+enum Paths {
+  static var modules: [String] {
+    ["SwiftBasicFormat", "SwiftIDEUtils", "SwiftParser", "SwiftSyntax", "SwiftSyntaxBuilder"]
+  }
+
+  static var packageDir: URL {
+    URL(fileURLWithPath: FileManager.default.currentDirectoryPath, isDirectory: true)
+      .deletingLastPathComponent()
+  }
+
+  static var sourcesDir: URL {
+    packageDir
+      .appendingPathComponent("Sources")
+  }
+
+  static var examplesDir: URL {
+    packageDir
+      .appendingPathComponent("Examples")
+  }
+
+  static var codeGenerationDir: URL {
+    packageDir
+      .appendingPathComponent("CodeGeneration")
+  }
+
+  static var workspaceDir: URL {
+    packageDir
```

Comment:
I think `workspaceDir` should be the folder containing `swift-syntax`. So this should be `packageDir.deletingLastPathComponent()`. And you don’t need to call `deletingLastPathComponent` in `llvmDir`.